### PR TITLE
ioredis-mock: Add cluster typing to ioredis-mock

### PR DESCRIPTION
**Pull request by ullumullu**
Expose `Cluster` option from ioredis-mock.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Cluster Option](https://github.com/stipsan/ioredis-mock/blob/main/src/index.js#L247)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ioredis-mock ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ioredis-mock@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ioredis-mock'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. Command
1. Cluster
1. once
1. on

<details>
<summary>as well as these 6 other properties...</summary>
<p>listenerCount, errorMonitor, captureRejectionSymbol, captureRejections, defaultMaxListeners, EventEmitter</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 37fb1e51f08fef54e379f84ec3118e5c96580659
</p>

File: types/ioredis-mock/index.d.ts [hunk 1]
```diff
@@ -8,15 +8,27 @@ import ioredis = require('ioredis');
 
 export type RedisOptions = { data?: Record<string, unknown> } & ioredis.RedisOptions;
 
+export type RedisClusterOptions = {
+    redisOptions: Omit<
```
Comment (ullumullu) [inline types/ioredis-mock/index.d.ts:12]: For reference this was taken from: https://github.com/luin/ioredis/blob/v5.3.2/lib/cluster/ClusterOptions.ts#L127

**Review by andrewbranch — APPROVED** [review]

**Comment by trim21** [comment]
Hi, @ullumullu do you have more comment to add? 

Looks like this pr is able to merge, you just need to reply “Ready to merge” to let bot merge it.

**Comment by ullumullu** [comment]
Ready to merge

**Pull request merged**